spec/factories/tags.rb

```diff
@@ -0,0 +1,15 @@
+# == Schema Information
+#
+# Table name: tags
+#
+#  id         :bigint           not null, primary key
+#  name       :string(256)      not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+FactoryBot.define do
+  factory :tag do
+    name { "Holidays" }
+    initialize_with { Tag.find_or_create_by(name:) }
```

Comment:
I'm not the biggest fan of quietly changing this from "create new tag" to "find existing tag"... certainly not in the base factory. I'd rather have helper methods to work with this if needed.